vpr/src/route/router_lookahead_extended_map.cpp

```diff
@@ -312,7 +312,7 @@ bool ExtendedMapLookahead::add_paths(RRNodeId start_node,
 
         if (size_t(this_node) != size_t(start_node)) {
             auto& parent_node = device_ctx.rr_nodes[size_t(parent)];
-            start_to_here = Entry(this_node, parent_node.edge_switch(paths[*it].edge), &start_to_here);
+            start_to_here = Entry(this_node, rr_graph.edge_switch(RREdgeId(paths[*it].edge)), &start_to_here);
```

Comment:
Look like you should use the ``edge_switch(RRNodeId, EdgeId)`` API here.

Reply:
I tried to use but I got errors.  Can you please suggest me how can we use?

Reply:
Sure. Can you put what you have tried? I can help.